source/D2Common/src/Items/ItemMods.cpp

```diff
@@ -655,59 +655,72 @@ BOOL __stdcall ITEMMODS_UpdateItemWithSkillCharges(D2UnitStrc* pItem, int nSkill
 }
 
 //D2Common.0x6FD928D0 (#10843)
-//TODO: Find names for variables
-int __stdcall D2COMMON_10843_GetByTimeAdjustment(int nAmount, int nTimeOfDay, int nBaseTime, int* a4, int* a5, int* a6)
+int __stdcall ITEMMODS_GetByTimeAdjustment(int nAmount, int nPeriodOfDay, int nBaseTime, int* nItemModPeriodOfDay, int* nItemModMin, int* nItemModMax)
 {
-	int v6;
-	int v7;
-	int v8;
-	int v9;
-	int v10;
+	// Time works in a cycle of 360 units.
+	constexpr int DAY_UNITS = 360;
 
-	v6 = ((nAmount >> 12) & 1023) - 256;
-	v7 = nAmount & 3;
-	v8 = ((nAmount >> 2) & 1023) - 256;
-
-	if (a4)
+	int nTempItemModPeriodOfDay = nAmount & 3;
```

Comment:
Could you take this opportunity to use hexadecimal values for the masks ? `0x3` here and `0x3FF` for the others